### PR TITLE
Merge tornjak components

### PR DESCRIPTION
**Pull request by maia-iyer**
experimental feature to solve #118

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,18 +21,19 @@ Otherwise, you can follow instructions below to build Tornjak images.
...
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) for the frontend build if running locally
 - [Golang](https://go.dev/doc/install) is required if you would like to run the Go backends locally
 
 ## Building Executables and Images
 
 Building Tornjak manually can be done with the Makefile. Notable make targets follow:
 - `make bin/tornjak-backend`: makes the Go executable of the Tornjak backend
 - `make bin/tornjak-manager`: makes the Go executable of the Tornjak manager
-- `make ui-agent`: makes the ReactJS app for the Tornjak frontend
+- `make ui-agent`: makes the optimized ReactJS app for the Tornjak frontend
```
Comment (mrsabath) [inline CONTRIBUTING.md:32]: I don't think it is an optimized version at this point.

File: USAGE.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 # Usage
 
-We support three container images currently:
+We publish four container images currently:
 - [Tornjak Backend](https://github.com/spiffe/tornjak/pkgs/container/tornjak-be): This image can be deployed as a sidecar with any SPIRE server. 
 - [Tornjak Manager](https://github.com/spiffe/tornjak/pkgs/container/tornjak-manager): A container that runs this image exposes a port to register multiple Tornjak backends and forward typical commands to multiple Tornjak backends from one API. 
 - [Tornjak Frontend](https://github.com/spiffe/tornjak/pkgs/container/tornjak-fe): This image is typically deployed after the Tornjak Backend or Manager are deployed, as it requires a URL to connect directly to the Tornjak backend API.  
+- [Tornjak](https://github.com/spiffe/tornjak/pkgs/container/tornjak): This image can deployed as one sidecare container alongside SPIRE that also includes the Tornjak frontend component. NOTE: this feature is experimental currently, but useful for quick deployment.
```
Comment (mrsabath) [inline USAGE.md:7]: Maybe we could rephrase this as:
"This image containing both Tornjak Backend and Frontend components can be deployed as a sidecar alongside SPIRE Server container"

And since this would be our new default, we can remove the NOTE, unless you think we should provide some kind of the warning?

**Review by mrsabath — APPROVED** [review]
Nice fix of the broken image! Thank you. Very small nits. LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (maia-iyer) [reply CONTRIBUTING.md:32]: ui-agent is - it creates local. Is not used for the container images though
